### PR TITLE
STRF-8769 Quick View - Image and price changes not reflected if an image is assigned to variant/sku

### PR DESCRIPTION
**Pull request by jairo-bc**
#### What?

Pressing on quick view, image changes not reflected

#### Tickets / Documentation

Add links to any relevant tickets and documentation.

- https://jira.bigcommerce.com/browse/STRF-8769
- https://jira.bigcommerce.com/browse/BCTHEME-295

### CODE REVIEW

File: templates/layout/base.html [hunk 1]
```diff
@@ -41,6 +41,9 @@
 
         {{{head.scripts}}}
 
+
+        {{~inject 'zoomSize' theme_settings.zoom_size}}
+        {{~inject 'productSize' theme_settings.product_size}}
```
Comment (BC-tymurbiedukhin) [inline templates/layout/base.html:46]: Could you please explain how moving injects helps to resolve this issue?

Comment (junedkazi) [reply templates/layout/base.html:46]: @jairo-bc can you please add some context as to why do we need to move them here instead on keeping them on product page.

Comment (jairo-bc) [reply templates/layout/base.html:46]: Basically, product.js (and product.html, where the variables are injected) is not being loaded on "quick view" click. So I put those variables on the upper level. 
cc @BC-tymurbiedukhin @junedkazi

**Review by BC-tymurbiedukhin — APPROVED** [review]
LGTM

**Comment by jairo-bc** [comment]
> LGTM, but could you please add changelog item for this pr? @jairo-bc

@BC-tymurbiedukhin done

**Comment by BC-tymurbiedukhin** [comment]
> @BC-tymurbiedukhin done

Could we move it into draft section?

**Pull request merged**